sim/humanoid_gym/envs/stompy_config.py

```diff
@@ -108,7 +108,7 @@ class sim(LeggedRobotCfg.sim):
 
         class physx(LeggedRobotCfg.sim.physx):
             num_threads = 12
-            solver_type = 1  # 0: pgs, 1: tgs
+            solver_type = 0  # 0: pgs, 1: tgs
```

Comment:
We should use tgs - can you run the training with this setup confirming it also works?